ChatQnA/README.md

```diff
@@ -344,3 +344,22 @@ OPEA microservice deployment can easily be monitored through Grafana dashboards
 
 ![chatqna dashboards](./assets/img/chatqna_dashboards.png)
 ![tgi dashboard](./assets/img/tgi_dashboard.png)
+
+## Tracing Services with OpenTelemetry Tracing and Jaeger
```

Comment:
I have no doubt about this PR. but shall we enable this feature on all examples and devices but not ChatQnA TGI only?

Reply:
will do if no concern for this PR.